spec.md

```diff
@@ -294,6 +294,13 @@ Deploy support is an OPTIONAL aspect of the Compose specification, and is
 described in detail in the [Deployment support](deploy.md) documentation.
 If not implemented the Deploy section SHOULD be ignored and the Compose file MUST still be considered valid.
 
+### attach
+
+When `attach` is defined and set to `false` a Compose implementation SHOULD not collect service logs,
```

Comment:
We should align this wording with the new tone, and remove `Compose implementation` and the uppercase `SHOULD`